### PR TITLE
PR: Algo Shell sort in java

### PR DESCRIPTION
**Pull request by sowmyasri-Z**
This PR is about shell sort in java

### CODE REVIEW

**Comment by sowmyasri-Z** [comment]
can u please check this?

File: 2-algorithms/1-java/shell-sort.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+# 🐚 SHELL SORT 🐚
+# DEFINITION 🥇
```
Comment (Abishethvarman) [inline 2-algorithms/1-java/shell-sort.md:2]: maintain a gap between the headings

**Review by Abishethvarman — APPROVED** [review]
Content seems good

**Comment by sowmyasri-Z** [comment]
modified

**Review by Abishethvarman — APPROVED** [review]
Approved to merge. @Thevapriyan07 please review fast and merge it.

**Review by Thevapriyan07 — APPROVED** [review]
APprove to merge

**Pull request merged**